flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/mailbox/MailboxExecutorImplTest.java

```diff
@@ -73,31 +74,30 @@ public void tearDown() {
 	}
 
 	@Test
-	public void testOperations() throws Exception {
-		final TestRunnable testRunnable = new TestRunnable();
-		mailboxExecutor.execute(testRunnable, "testRunnable");
-		Assert.assertEquals(testRunnable, mailbox.take(DEFAULT_PRIORITY).getRunnable());
+	public void testPutGet() throws Exception {
```

Comment:
why is it called `testPutGet`? 

`testExecuteSingleMail`?

Reply:
Because what actually being tested here is `put` and `get` operations on the `mailbox`.

Reply:
But you are not invoking those methods anywhere in this test, and they are a private implementation detail of the `MailboxExecutorImpl` - the class that it's actually being tested here.

Reply:
returned previous name `testOperations()`